BitwardenShared/UI/Platform/Application/AppProcessor.swift

```diff
@@ -380,6 +371,47 @@ extension AppProcessor {
         }
     }
 
+    /// Handle a "bitwarden://" url.
+    ///
+    /// - Parameter url: The Bitwarden URL received by the app.
+    ///
+    private func handleBitwardenUrl(url: URL) async {
+        guard let components = URLComponents(url: url, resolvingAgainstBaseURL: true) else { return }
+
+        if components.host == "settings", components.path == "/account_security" {
```

Comment:
🎨 Could it be added to the `guard` instead?
```suggestion
        guard let components = URLComponents(url: url, resolvingAgainstBaseURL: true),
                   components.host == "settings", components.path == "/account_security" else { 
            return 
        }
```

Reply:
The only reason I didn't use a `guard` in the first place is I am planning to add at least one more route in a future PR. So it was cleaner to split it into components and then plan on taking multiple paths. But it was so much cleaner for now as a `guard` I just switched it to a `guard` in the latest. I'll refactor as needed in the future PR where I need to. 👍